spmt/spmt-ps/spmt/Register-SPMTMigration.md

```diff
@@ -270,7 +270,7 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-### -MigrateHiddenFiles
+### -IncludeHiddenFiles
 If set to On, hidden system files will be migrated.
```

Comment:
This is a bool. I'd recommend using true/false or $true/$false instead of On/Off.